### DISCUSSION TITLE
Valutwarden not comaptible with Bitwarden client application v2022.11.0 (Mac OS - AppStore)

### DISCUSSION THREAD

**Discussion by makovice** [dani-garcia/vaultwarden#3018] [Q&A] [answered]
### Subject of the issue
Prelogin endpoint (/identity/accounts/prelogin) does not work. 

### Deployment environment

* vaultwarden version: vaultwarden/server:testing (1.26.0-10dadfca)

* Install method: Docker image

* Clients used: Mac OS desktop 2022.11.0, Architecture arm64

### Steps to reproduce

When connecting from client app, found in webserever log

   ... "GET /api/devices/knowndevice/xxx@xxxx.cz/xxxxxxxxxxxxxxx HTTP/2.0" 404 597 "-" ...
   ... "POST /identity/accounts/prelogin HTTP/2.0" 404 597 "-" ...
   ... "POST /identity/connect/token HTTP/2.0" 400 227 "-" ...

When revert to old client app (found v2022.5.0)

   ... "POST /api/accounts/prelogin HTTP/2.0" 200 32 "-" ...
   ... "POST /identity/connect/token HTTP/2.0" 200 3397 "-" ...

### Expected behaviour
So maybe only changed endpoint from /api/... to /identity/... ???

**Comment by BlackDex** [comment, marked as answer]
Already fixed in `main` branch and `testing` tagged docker images.

**Reply by makovice** [reply]
I tried to update "docker-compose pull" but no change :( ... I found that have version 1.26.0-10dadfca. Is it last?

in docker-compose.yml ... image: vaultwarden/server:testing

**Reply by BlackDex** [reply]
Did you do a `docker-compose up -d` afterwards?

**Reply by makovice** [reply]
No :( ... sorry I am not friendly with docker :D I am little more old-school programmer :D :D 

It is working now.

Thank you very much and please close this.

**Reply by BlackDex** [reply]
Glad is fixed!

**Reply by denysvitali** [reply]
Tested w/ the Android mobile app and the `testing` image:
```
POST /identity/connect/token => 400 Bad Request
```

Am I missing something?

**Reply by BlackDex** [reply]
`docker-compose up -d`?

**Reply by denysvitali** [reply]
I've deployed Vaultwarden on Kubernetes and using the latest version:
```
./vaultwarden -v
vaultwarden 1.26.0-10dadfca
```

**Reply by makovice** [reply]
Bad request is sure other problem, then missing endpoint (error 404) ;)

**Reply by BlackDex** [reply]
That should work just fine, I'm running that version and my Android can login fine.

Check the reverse proxy log and config, if there is nothing blocking or something? Maybe it mangels the request? Try debug log_level, or trace, and see what happens.

**Reply by denysvitali** [reply]
Nevermind, user's fault :D
I tried to debug locally and it was failing in the login part (I though they changed something in the way the parameters are being sent to the server on the Android app) but... I was using the wrong master password, of course :facepalm:
